crontab

```diff
@@ -0,0 +1,4 @@
+# start cron
+*/10 * * * * rm -rf /home/ga/static/tmp/* && echo "`date +\%Y\%m\%d-\%H:\%M:\%S` tmp folder cleared" >> /var/log/cron.log 2>&1
```

Comment:
is this set for 10am daily? I hate reading cron things as I never remember syntax

Reply:
that was for testing, it ran every 10 minutes. Changed back to 2am daily 0 2 * * *